actions/scan-packages/action.yaml

```diff
@@ -24,7 +33,28 @@ runs:
         else
           echo "::error::Couldn't find a script section in package.json"
         fi
-
     - name: Verify stardust
       shell: bash
       run: pnpm -r --workspace-concurrency=1 exec ${{ github.action_path }}/checkPeerDeps.sh
+    - name: Get repo matrix
```

Comment:
This job should output the matrix instead of three different arrays. Since we wan't to run specific configurations we should use include (second example in this [section](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#example-adding-configurations))

For nebula-tables I think the output should look like this
```
{
  "fail-fast": false,
  "matrix": [
    "include": [
      {
        "package": "table-utils",
        "folder": "packages/table-utils"
      },
      {
        "package": "sn-table",
        "folder": "packages/sn-table",
        "systemjsFiles": ["file1","file2"]
      },
      {
        "package": "sn-pivot-table",
        "folder": "packages/sn-pivot-table",
        "systemjsFiles": ["file1","file2"]
      }
    ]
  }
}
```